linter.py

```diff
@@ -15,7 +15,15 @@
 import shutil
 import tempfile
 
-from SublimeLinter.lint import Linter, util, highlight, persist
+from SublimeLinter.lint import Linter, util, persist
+
+import SublimeLinter

```

Comment:
Technically, this should be `import SublimeLinter.lint`, but it will work either way since we already imported from that module earlier.

Reply:
Yeah, I actually didn't think very hard about this but just copy-pasted it around for a couple plugins. Adjust any way you like.